src/view_control.cpp

```diff
@@ -369,7 +369,7 @@ void View::DrawBracketSpan(
         dc->ResumeGraphic(graphic, graphic->GetUuid());
     }
     else {
-        dc->StartGraphic(bracketSpan, "spanning-bracketspan", "");
+        dc->StartGraphic(bracketSpan, "spanning-bracketspan", bracketSpan->GetUuid(), false);
```

Comment:
Please remove the `-` for all spanning graphics as suggested elsewhere

Reply:
Done